public/css/styles.css

```diff
@@ -150,7 +150,7 @@
     color: rgba(255, 255, 255, 0.75);
   }
   header a:hover {
-    color: rgb(255, 255, 255);
+    color: white;
```

Comment:
Have you done `npm install` recently? `sass` has been updated recently, which swaps named colors out for the `rgb` form.

Reply:
See this: https://github.com/playcanvas/developer.playcanvas.com/commit/dd995d0a9260f74b65b31f3876ac78029d3e160f